### PR TITLE
swap azure accounts to track azure test anomaly on this job

### PR DESCRIPTION
**Pull request by DennisPeriquet**

### CODE REVIEW

File: ci-operator/config/openshift/release/openshift-release-master__ci-4.10-upgrade-from-stable-4.9.yaml [hunk 1]
```diff
@@ -36,9 +36,8 @@ tests:
 - as: e2e-azure-upgrade
   interval: 24h
   steps:
-    cluster_profile: azure-2
+    cluster_profile: azure-4
```
Comment (petr-muller) [inline ci-operator/config/openshift/release/openshift-release-master__ci-4.10-upgrade-from-stable-4.9.yaml:39]: `azure4`

**Review by petr-muller — COMMENTED** [review]
One change, plus this will also need `make jobs`

**Comment by DennisPeriquet** [comment]
ok, I see all I need to do it go to the root dir in this repo and do a `make jobs` on a machine with docker/podman and make.  I can do this once this merges.

**Pull request closed without merging**